src/notation/inotationinteraction.h

```diff
@@ -194,6 +194,9 @@ class INotationInteraction
     virtual void toggleBold() = 0;
     virtual void toggleItalic() = 0;
     virtual void toggleUnderline() = 0;
+
+    virtual void loadStyle(const mu::io::path&) = 0;
+    virtual void saveStyle(const mu::io::path&) = 0;
```

Comment:
Better to move those methods to INotationStyle

Reply:
Are you sure? For loadStyle() need the startEdit()/apply()/notifyAbout... functions from NotationInteraction too.
I'd agree it might be a good place to put the "selectStyleFile" function that shows the load/save dialog for style files.

Reply:
You can pass NotationUndoStack to NotationStyle:

<img width="917" alt="Screenshot 2021-08-18 at 09 41 28" src="https://user-images.githubusercontent.com/67867444/129858237-66945501-9a3d-48bf-beaa-0b316337a19a.png">

But I don't understand why you need notifyAboutSelectionChanged in loadStyle. Could you explain please?

Reply:
So the properties panel can update itself as necessary.

Reply:
Better to use styleChanged() in this case, because we can't guarantee that selection is really changed

Reply:
True, assuming the properties panel is listening to that! (obviously it should be)

Reply:
Actually it wasn't, but have fixed.

Reply:
Done